modules/installation-uninstall-clouds.adoc

```diff
@@ -24,6 +27,12 @@ ifdef::aws[]
 If you deployed your cluster to the AWS C2S Secret Region, the installation program does not support destroying the cluster; you must manually remove the cluster resources.
 ====
 endif::aws[]
+ifdef::gcp[]
+[NOTE]
+====
+After uninstallation, please check your cloud provider for any resources not removed properly, especially with User Provisioned Infrastructure (UPI) clusters. There may be resources that were not created by the installer or that the installer doesn't have access to. For instance, Google Cloud resources that require link:https://cloud.google.com/iam/docs/overview#concepts_related_to_access_management[IAM permissions] in shared vpc host project and unused link::https://cloud.google.com/sdk/gcloud/reference/compute/health-checks/delete[health checks that need to be deleted].
```

Comment:
Per IBM SG:

* Remove "please"
* s/may/might/
* s/doesn't/does not/
* s/For instance/For example/
* s/need to be/must be/ (<-- if this is technically accurate / not too restrictive)

Reply:
The last sentence seems like a sentence fragment. Suggest something like:

```
For example, some Google Cloud resources require link:https://cloud.google.com/iam/docs/overview#concepts_related_to_access_management[IAM permissions] in shared VPC host projects and unused link:https://cloud.google.com/sdk/gcloud/reference/compute/health-checks/delete[health checks that must be deleted].
```

Note also in the above:

* s/vpc/VPC/
* s/project/projects/
* The second `link:` had 2 colons in the original (`link::`)